### PR TITLE
BatchInvoke, batch data loaders on datasource.type.field

### PR DESCRIPTION
**Pull request by JimmyHurrah**
Discussion here https://github.com/bboure/serverless-appsync-simulator/pull/29

### CODE REVIEW

File: src/data-loaders/LambdaDataLoader.js [hunk 1]
```diff
@@ -9,6 +9,17 @@ const getBatchDataResolver = (loaderName, resolver) => {
   return batchLoaders[loaderName];
 };
 
+const getLoaderName = (payload, config) => {
+  const parts = [config.name];
+  if (payload && payload.info && payload.info.parentTypeName) {
```
Comment (bboure) [inline src/data-loaders/LambdaDataLoader.js:14]: @JimmyHurrah You can use the shorthand

```suggestion
  if (payload?.info?.parentTypeName) {
```

Comment (JimmyHurrah) [reply src/data-loaders/LambdaDataLoader.js]: Ah yeah, ofc, I was unsure if it was supported, I forgot it got compiled with babel 👍

**Comment by bboure** [comment]
LGTM THanks!

**Comment by bboure** [comment]
Published in v0.5.1 🎉

**Pull request merged**